### PR TITLE
PLANNER-707: Workbench "import example dialog": Embed stock example repositories

### PR DESCRIPTION
**Pull request by mcimbora**
* Remove kie-wb-example-repositories module
* Add dependency on org.kie.workbench.playground:playground-parent

Depends on https://github.com/droolsjbpm/kie-wb-common/pull/622

### CODE REVIEW

File: kie-drools-wb/kie-drools-wb-webapp/src/main/java/org/kie/workbench/drools/backend/server/AppSetup.java [hunk 1]
```diff
@@ -76,34 +70,11 @@ public AppSetup( @Named("ioStrategy") final IOService ioService,
     }
 
     @PostConstruct
-    public void assertPlayground() {
+    public void init() {
         try {
             configurationService.startBatch();
-            final String exampleRepositoriesRoot = System.getProperty( "org.kie.example.repositories" );
```
Comment (jhrcek) [inline kie-drools-wb/kie-drools-wb-webapp/src/main/java/org/kie/workbench/drools/backend/server/AppSetup.java]: If `org.kie.example.repositories` and `org.kie.example` are going to be removed (and not reintroduced somewhere else) please make sure, that it's also removed from kie-docs to keep it in sync

Comment (mcimbora) [inline kie-drools-wb/kie-drools-wb-webapp/src/main/java/org/kie/workbench/drools/backend/server/AppSetup.java]: @jhrcek +1

**Comment by mcimbora** [comment]
Jenkins retest this

**Comment by manstis** [comment]
The test failures are related to these PRs (as the test ```ProjectAuthoringIntegrationTest.importAndBuildProject()``` imports a repository using the Wizard and builds the relating project. The UI elements of course have all changed with these PRs).

**Comment by mcimbora** [comment]
Jenkins retest this

**Comment by mcimbora** [comment]
@manstis Fixed the broken test to match current implementation.

**Comment by manstis** [comment]
@mcimbora Thanks.

**Pull request merged**